### DISCUSSION TITLE
Link Label Font Size

### DISCUSSION THREAD

**Discussion by ndtreviv** [f5/unovis#112] [Q&A] [answered]
I'm using a link label function like so:

```
  const linkLabel = useCallback((l) => {
    switch (l.type) {
      case 'A': return { text: 'Affiliated With' };
      case 'B': return { text: 'Owned By' };
      case 'C': return { text: 'Resolves To' };
      default: return { text: l.type || 'UNKNOWN' };
    }
  }, []);
```

And passing it as a property on my `<VisGraph ...>` object like so:
```
...
   <VisGraph
      linkLabel={linkLabel}
     ...
```

The link text is teeny tiny. According to the docs the function should return a `GraphCircleLabel` object, which looks like this:
```
type GraphCircleLabel = {
  text: string;
  color?: string | null; // Optional color
  cursor?: string | null; // Optional cursor on hover
  fontSize?: string; // Optional font size as a CSS string
  radius?: number; // Circle radius in pixels
}
```

So I figured I could increase the link text size slightly by setting the `fontSize` property, like so:

```
  const linkLabel = useCallback((l) => {
    switch (l.type) {
      case 'A': return { text: 'Type A', fontSize: '2rem' };
      case 'B': return { text: 'Type B' };
      case 'C': return { text: 'Type C' };
      default: return { text: l.type || 'Unknown' };
    }
  }, []);
```
But that doesn't work. The label of the `Type A` link is no different from the rest. I've also tried putting px values in there amongst some other stuff.

What am I doing wrong?

**Comment by rokotyan** [comment, marked as answer]
@ndtreviv This is definitely a bug, thanks for reporting! I'm planning to fix it this within the next few days.

**Reply by rokotyan** [reply]
@ndtreviv Unovis `1.0.3-beta.1` with the fix has just been published.
